tensorflow/c/c_api.cc

```diff
@@ -1696,6 +1696,8 @@ TensorId ToTensorId(const TF_Output& output) {
 
 }  // namespace
 
+const int TF_CONTROL_SLOT = tensorflow::Graph::kControlSlot;
```

Comment:
Likewise move this closer to the TF_Operation section (maybe after the TF_Session definition?)